org.graalvm.truffle.truffle-runtime/osgi.bnd

```diff
@@ -3,12 +3,8 @@ Bundle-Name: ${project.name}
 Bundle-License: Universal Permissive License, Version 1.0
 Bundle-Version: ${project.version}
 Import-Package: \
-  jdk.internal.access;resolution:="optional", \
```

Comment:
Aren't those needed anymore? I needed to add them when initialisation OSGi-ifying the bundles.